services/injective.ts

```diff
@@ -90,11 +90,12 @@ class Injective {
     network: InjectiveNetwork = InjectiveNetwork.TestnetK8s
   ) {
     const endpoints = getNetworkEndpoints(network)
-
     this.offlineSigner = offlineSigner
-    this.txClient = new TxRestClient(endpoints.rest)
+    this.txClient = new TxRestClient('https://ww-injective-rest.polkachu.com/')
```

Comment:
Because this is our custom endpoint no issue for now I remember Injective was tough to make work so maybe this is needed but in future we should switch back to something dynamic